articles/vmware/vmw-how-convert-edge.md

```diff
@@ -43,6 +43,9 @@ Before converting your edge gateway, review the prerequisites and known issues b
 
 - **Edge gateway version** --- To convert to an advanced gateway, your edge gateway must be at version 6.2.*x*. If necessary, redeploy your edge gateway to upgrade to the latest version; for more information,
     see [*How to upgrade your edge gateway*](vmw-how-upgrade-edge.md).
+    
+> [!IMPORTANT]
```

Comment:
Indent this line and the next by four spaces so that the note is aligned correctly with the bullet point.

Reply:
Done. I didn't realise it retained leading spaces.